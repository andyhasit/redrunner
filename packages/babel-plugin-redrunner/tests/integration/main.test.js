// tests/snapshots/index-test.js
const {c, EOL, transform} = require('../utils')

const src = `
class MyView extends View {
  __html__ = '<div/>'
}
`

it('works', () => {
  const code = transform(src)
  expect(code).toMatchSnapshot()
});