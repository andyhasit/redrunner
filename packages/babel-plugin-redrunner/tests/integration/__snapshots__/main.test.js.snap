// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
"class MyView extends View {}

MyView.prototype.__wc = [];
MyView.prototype.__ht = '<div></div>';

MyView.prototype.__bv = function (view, prototype) {
  view.__bd(prototype, false);

  view.dom = {};
};"
`;
